tensorbay/client/version.py

```diff
@@ -756,10 +756,18 @@ def get_job(self, job_id: str) -> SquashAndMergeJob:
         Arguments:
             job_id: The SquashAndMergeJob id.
 
-        Return:
+        Returns:
             The SquashAndMergeJob.
 
         """
+        job_info = self._get_job(job_id)
+        job = SquashAndMergeJob.loads(job_info)
+
+        job._job_updater = self._get_job  # pylint: disable=protected-access
+        job._dataset_id = self._dataset_id  # pylint: disable=protected-access
+        job._client = self._client  # pylint: disable=protected-access
```

Comment:
why these private members is not set in `create_job`?

Reply:
Yes, the create_job and list_job  also needs to set these private members. So is it necessary to set up a new function to do this in JobMixin? But that would be a little weird.
```
def _update_attrs(self, job):
    job._job_updater = self._get_job
    job._dataset_id = self._dataset_id
    job._client = self._client
```